profiles/kentik_snmp/dell/sonicwall-sma.yml

```diff
@@ -0,0 +1,222 @@
+# https://mibs.observium.org/mib/SNWL-COMMON-MIB/
+# https://mibs.observium.org/mib/SONICWALL-SMA-APPLIANCE-SYSTEM-HEALTH-MIB/
+# https://mibs.observium.org/mib/SONICWALL-SMA-APPLIANCE-SERVICE-HEALTH-MIB/
+# https://mibs.observium.org/mib/SONICWALL-SMA-APPLIANCE-SECURITY-HISTORY-MIB/
+# https://mibs.observium.org/mib/SONICWALL-SMA-APPLIANCE-TUNNEL-SERVER-MIB/
+---
+extends:
+  - system-mib.yml
+  - if-mib.yml
+  - host-resources-mib.yml
+
+provider: kentik-firewall
+
+sysobjectid:
+  - 1.3.6.1.4.1.8072.3.2.10.sonicwall-sma
```

Comment:
@i3149 - can you confirm the hyphen in the suffix here won't break anything please?

Reply:
Hyphen should be fine.

Reply:
In my testing:

```
2022-03-18T12:20:33.967 ktranslate(250350) [Info] KTranslate Client SNMP: Running SNMP for bart on 10.10.0.1 (type=kentik-net-snmp)
2022-03-18T12:20:33.967 ktranslate(250350) [Info] KTranslate>bart Found profile for .1.3.6.1.4.1.8072.3.2.8: sonicwall-sma.yml net-snmp.yml
2022-03-18T12:20:33.967 ktranslate(250350) [Info] KTranslate>bart Setting profile of kentik-firewall for device bart.
```